### DISCUSSION TITLE
2021/12/25/Test-uncompressed-assets-with-Robolectric

### DISCUSSION THREAD

**Discussion by giscus** [utzcoz/utzcoz.github.io#21] [Comments]
# 2021/12/25/Test-uncompressed-assets-with-Robolectric

This article is very short and just used to show how to test uncompressed assets with Robolectric. What we need do is very simple:

https://utzcoz.github.io/2021/12/25/Test-uncompressed-assets-with-Robolectric.html

**Comment by utzcoz** [comment]
Test comment: Looks like this blog can receive comments from other folks now.

**Reply by utzcoz** [reply]
Test response from GitHub Discussion directly.

**Comment by utzcoz** [comment]
Test comment: this comment is posted from GitHub Discussion directly.
